docs/installation_guide/installation.rst

```diff
@@ -158,7 +162,11 @@ Here’s the list of the singer connectors and if they are installed by default
 +----------------------------+-----------------------------------------+----------------------------------+---------------------------------------+
 | tap-zendesk                | ./install --connectors=tap-zendesk      | YES                              |                                       |
 +----------------------------+-----------------------------------------+----------------------------------+---------------------------------------+
-| target-postgres            | ./install --connectors=target-postgres  | NO                               |                                       |
+| tap-zuora                  | ./install --connectors=tap-zendesk      | NO                               |                                       |
```

Comment:
typo in `--connectors`

Reply:
thanks, fixed by https://github.com/transferwise/pipelinewise/pull/473/commits/c960e22a380e4f4bb08997609b292b812f86bb0f